fdpp/thunks.cc

```diff
@@ -366,7 +367,19 @@ void fdprintf(const char *format, ...)
     va_end(vl);
 }
 
-void fdlogprintf(const char *format, ...)
+void fdprintf(const char *format, ...)
```

Comment:
I don't think we need to re-do fdprintf()
that way. fdprintf() is for our own debugging,
say, in thunks.cc, so it should be compatible
with dosemu.
I meant the following:
```
You should define the new function that
will to _sprintf() into the buffer, and then
pass it to fdprintf() already with things
like %P expanded.
```

Also are you sure you need to re-define
`DebugPrintf()`? You were saying this:
`After getting tn_printf() in truename to print to dosemu I find that dosemu crashes.`
Do you really need to redefine anything
more that `tn_printf()`? `DebugPrintf()`
prints to dosemu terminal anyway, or
do you want into the log?

Reply:
At the time %P was being passed through to dosemu, which was ignored as it doesn't know how to deal with it, so the following parameters didn't match the format passed.  I believe that caused the crash.

Reply:
Yes, but why to re-define `fdprintf()` for this?
In particular, why to even want to print debug
via fdpp API, if they go to dosemu screen anyway?